lib/jekyll-github-metadata/repository_finder.rb

```diff
@@ -46,15 +48,19 @@ def nwo_from_config
       end
 
       def git_exe_path
+        exts = (ENV["PATHEXT"] || "").split(File::PATH_SEPARATOR).push("")
+        cmds = exts.map { |ext| "git#{ext}" }
         ENV["PATH"].to_s
           .split(File::PATH_SEPARATOR)
-          .map { |path| File.join(path, "git") }
-          .find { |path| File.exist?(path) }
+          .map { |path| cmds.map { |cmd| File.join(path, cmd) } }
+          .flatten
+          .find { |path| File.executable?(path) && !File.directory?(path) }
```

Comment:
Is this branching really necessary?
When is an executable ever a directory?

Reply:
Directories are always executable.  So if there is a `git` directory in `PATH`, we don't want to select it.

Windows:
```
irb(main):006:0> File.directory?('/Users')
=> true
irb(main):007:0> File.executable?('/Users')
=> true
```

Linux:
```
irb(main):002:0> File.directory?('/home')
=> true
irb(main):003:0> File.executable?('/home')
=> true
```

Reply:
Today I Learned <sup>TM</sup> :+1: